src/integrationTest/java/uk/gov/hmcts/reform/roleassignment/config/SwaggerPublisher.java

```diff
@@ -20,27 +20,31 @@
  * Built-in feature which saves service's swagger specs in temporary directory.
  * Each travis run on master should automatically save and upload (if updated) documentation.
```

Comment:
Can we update to refere to workflow rather than travis: .github/workflows/swagger.yml

Reply:
comments updated

Reply:
Resolved in e98225bd1c69790afa414a0ddf58d3d65cb3cead